tests/integration-tests/tests/cloudwatch_logging/cloudwatch_logging_boto3_utils.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+
+import utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_log_groups():
+    """Get list of log groups."""
+    logs_client = boto3.client("logs")
+    log_groups = logs_client.describe_log_groups().get("logGroups")
+    LOGGER.debug("Log groups: {0}\n".format(json.dumps(log_groups, indent=4)))
+    return log_groups
+
+
+def get_log_streams(log_group_name):
+    """Get list of log streams."""
+    logs_client = boto3.client("logs")
+    streams = logs_client.describe_log_streams(logGroupName=log_group_name).get("logStreams")
+    LOGGER.debug(
+        "Log streams for {group}:\n{streams}".format(group=log_group_name, streams=json.dumps(streams, indent=4))
+    )
+    return streams
+
+
+def get_log_events(log_group_name, log_stream_name):
+    """Get log events for the given log_stream_name."""
+    logs_client = boto3.client("logs")
+    events = logs_client.get_log_events(logGroupName=log_group_name, logStreamName=log_stream_name).get("events")
+    LOGGER.debug(
+        "Log events for {group}/{stream}:\n{events}".format(
+            group=log_group_name, stream=log_stream_name, events=json.dumps(events, indent=4)
+        )
+    )
+    return events
+
+
+def get_ec2_instances():
+    """Iterate through ec2's describe_instances."""
+    ec2_client = boto3.client("ec2")
+    paginator = ec2_client.get_paginator("describe_instances")
+    for page in paginator.paginate():
```

Comment:
We have a `paginate_boto3` function in the `pcluster.utils` we can define the same in the tests/utils